ADALiOS/ADALiOS/ADHelpers.m

```diff
@@ -245,5 +246,39 @@ +(uint8_t*) updateDataInput:(uint8_t) ctr
     return tmpFixedInput;
 }
 
++ (NSURL*)addClientVersionToURL:(NSURL*)url
+{
+    if (!url)
```

Comment:
It will be better to have single line or use parentheses.

Other lines also need modification for that.